sdk/storage/Azure.Storage.Common/tests/Shared/StorageTestBase.cs

```diff
@@ -49,14 +49,14 @@ public void SetUp()
         {
             if (Debugger.IsAttached || Mode == RecordedTestMode.Live)
             {
-                _listener = new TestEventListener();
+                s_listener = new TestEventListener();
             }
         }
 
         [OneTimeTearDown]
         public void CleanUp()
```

Comment:
Slight nit with these `SetUp` and `CleanUp` methods - these names (at least `SetUp`) are semi-likely to be used in derived classes if they want their own setup/teardown.  Could we rename them to something like `StartLoggingTestOutput` and `StopLoggingTestOutput` or similar?